### DISCUSSION TITLE
Ms 2021 white screen

### DISCUSSION THREAD

**Discussion by Get-108** [tesla-android/issue-tracker#309] [Q&A] [answered]
Hello!
Started to show white screen in browser today at device.teslaandroid.com.
After a short tesla android logo at start.
Wifi connected.

**Comment by mikegapinski** [comment, marked as answer]
Hi,

The SSL certificate has expired. Please use the http:// website and update to Tesla Android 2023.45.1. This version also improves compatibility with Tesla firmware 2023.38.6 or newer.

Best regards

Od: Get-108 ***@***.***>
Data: wtorek, 14 listopada 2023 o 17:39
Do: tesla-android/issue-tracker ***@***.***>
DW: Subscribed ***@***.***>
Temat: [Tesla Android] Ms 2021 white screen (Discussion #309)

Hello!
Started to show white screen in browser today at device.teslaandroid.com.
After a short tesla android logo at start.
Wifi connected.

**Comment by Get-108** [comment]
Thanks! It works!
